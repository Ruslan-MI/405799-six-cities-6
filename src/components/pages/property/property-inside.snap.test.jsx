import React from "react";
import {
  render,
} from "@testing-library/react";
import PropertyInside from "./property-inside";
import {
  mockOffer,
} from "../../../mocks-for-test";

it(`Property inside should render correctly`, () => {

  const {
    container,
  } = render(<PropertyInside goods={mockOffer.goods} />);

  expect(container).toMatchSnapshot();
});
