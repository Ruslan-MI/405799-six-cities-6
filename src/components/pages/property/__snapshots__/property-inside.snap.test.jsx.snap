// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Property inside should render correctly 1`] = `
<div>
  <div
    class="property__inside"
  >
    <h2
      class="property__inside-title"
    >
      What's inside
    </h2>
    <ul
      class="property__inside-list"
    >
      <li
        class="property__inside-item"
      >
        Heating
      </li>
      <li
        class="property__inside-item"
      >
        Kitchen
      </li>
      <li
        class="property__inside-item"
      >
        Cable TV
      </li>
      <li
        class="property__inside-item"
      >
        Washing machine
      </li>
      <li
        class="property__inside-item"
      >
        Coffee machine
      </li>
      <li
        class="property__inside-item"
      >
        Dishwasher
      </li>
    </ul>
  </div>
</div>
`;
